sleap/gui/commands.py

```diff
@@ -2029,6 +2033,34 @@ def ask(context: CommandContext, params: dict) -> bool:
 
         return len(filename) > 0
 
+class LinkVideoToSession(EditCommand):
+    topics = [UpdateTopic.video]
+
+    @staticmethod
+    def do_action(context: CommandContext, params: dict):
+        video = context.state["video"]
+        session = context.state["session"]
+
+        if video is None:
+            raise ValueError("No video selected.")
+
+        if session is None:
+            raise ValueError("No session selected.")
+
+        video.recording_session = session
+
```

Comment:
Lets also reset the selected video and camera here:
```suggestion

        # Reset the selected camera and video
        context.state["selected_camera"] = None
        context.state["selected_unlinked_video"] = None
```